regex-syntax/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "regex-syntax"
 version = "0.6.2"  #:version
 authors = ["The Rust Project Developers"]
-license = "MIT/Apache-2.0"
+license = "MIT/Apache-2.0/Unicode"
```

Comment:
I'm not sure that's entirely appropriate, since it isn't really referring to the license of the crate as a whole. With your change, it makes it look like `regex-syntax` is triply licensed, which I don't think is accurate. Specifically, I think

> this copyright and permission notice appear in associated Documentation

is satisfied by putting the Unicode license in this repo and adding a note to the README.

Reply:
Yeah I suppose that's true. I'll remove it.